demisto_sdk/commands/content_graph/strict_objects/common.py

```diff
@@ -49,6 +49,13 @@ def prevent_none(cls, value, field):
             "playbook_id",
             "query",
             "playbook_input_query",
+            "suppression_duration",
+            "suppression_fields",
+            "user_defined_category",
+            "user_defined_severity",
+            "investigation_query_link",
+            "cron_tab",
+            "search_window",
```

Comment:
```suggestion
            "suppression_duration", # correlation rules
            "suppression_fields", # correlation rules
            "user_defined_category", # correlation rules
            "user_defined_severity", # correlation rules
            "investigation_query_link", # correlation rules
            "cron_tab", # correlation rules
            "search_window", # correlation rules
```
to make it easier on our future selves to know why we added it here